docs/index.md

```diff
@@ -1 +1,100 @@
-# TODO
+## About
+
+**Blender Image Preview**, or BIP for short is a **[Blender](https://blender.org)**
+library by 3D Ninjas that allows addon developers to have extra functionality
+that comes built into Blender but takes quite some time to set up.
+
+So what exactly is the `.bip` format? The accronym is exactly the same as the
+name of the library **Blender Image Preview**. Its a format designed to be 100%
+compatible with the Blender in-memory format of these objects. It primarily
+guarantees, that we can just read the data from the file and copy it without any
+modifications in the right buffer.
+
+### Origin
+
+BIP came about as a solution to image previews on an asset manager that we are
+currently developing at **[3D Ninjas](https://3dninjas.io)**. While working on
+this project, we realized that the resolution of the images provided by the
+default `bpy.utils.previews` was quite limiting in that we had no control as to
+image quality/resolution. After some sessions and development, BIP came to life.
+We decided that such knowledge should not be hoarded but rather shared with the
+Blender community.
+
+### Ease-Of-Use
+
+The library can be used as a direct drop-in replacement for the Blender's
+default `bpy.utils.previews` module, however, it pays off to take a bit of time
+to look into the other features that come with the library as they are what make
+it stand out.
+
+---
+
+## Use Cases
+
+### Controlled Environment
+
+Quickly load optimized images images that they have control over into Blender
```

Comment:
More along the line "In case you create all images yourself, you can use the highly optimized BIP format for the images. In that case Pillow is not required on the users computer."

Reply:
Updated.